### PR TITLE
zh-cn for Scala Tour: inner-classes.md

### PR DESCRIPTION
**Pull request by realwunan**
Simplified Chinese version of Scala Tour: Inner Classes

### CODE REVIEW

File: _zh-cn/tour/inner-classes.md [hunk 1]
```diff
@@ -13,3 +13,70 @@ language: zh-cn
 next-page: abstract-types
 previous-page: lower-type-bounds
 ---
+
+在Scala中，一个类可以作为另一个类的成员。 在一些类似 Java 的语言中，内部类是外部类的成员，而 Scala 正好相反，内部类是绑定到外部对象的。 假设我们希望编译器在编译时阻止我们搞乱节点 nodes 与图形 graph 的关系。 路径依赖的类型提供了一种解决方案。
```
Comment (liufengyun) [inline _zh-cn/tour/inner-classes.md]: `搞乱` -> `混淆`?

`。` -> `，`

`路径依赖的类型` -> `路径依赖类型`?

File: _zh-cn/tour/inner-classes.md [hunk 2]
```diff
@@ -13,3 +13,70 @@ language: zh-cn
...
+
+为了说明差异，我们简单描述了图形 graph 数据类型的实现：
```
Comment (liufengyun) [inline _zh-cn/tour/inner-classes.md]: `一个图形数据类型`

File: _zh-cn/tour/inner-classes.md [hunk 3]
```diff
@@ -13,3 +13,70 @@ language: zh-cn
...
+  }
+  var nodes: List[Node] = Nil
+  def newNode: Node = {
+    val res = new Node
+    nodes = res :: nodes
+    res
+  }
+}
+```
+该程序将图形表示为节点列表 (`List[Node]`)。 每个节点都有一个用来存储与其相连的其他节点的列表 (`connectedNodes`)。 类 `class Node` 是一个 _路径依赖的类型_，因为它嵌套在类 `class Graph` 中。 因此，`connectedNodes` 中存储的所有节点必须使用同一个 `Graph` 的实例对象的 `newNode` 方法来创建。
```
Comment (liufengyun) [inline _zh-cn/tour/inner-classes.md]: 类 `class Node` --> 类`Node` ?

`路径依赖的类型`  -> `路径依赖类型`

类 `class Graph` -> 类 `Graph`

Comment (realwunan) [reply _zh-cn/tour/inner-classes.md]: Thanks a lot. 
类 `class Graph` 是根据英文原文来的，的确不妥。

**Comment by liufengyun** [comment]
Thanks a lot, @realwunan 🎉

**Pull request merged**